script.js

```diff
@@ -0,0 +1,602 @@
+            }else if(englishKeysUp[key] === "&#8594"){
+                button.classList.add("key");
+                button.classList.add("ArrowRight");
+            }else if(russianKeys[key] == "Backspace"){
+                button.classList.add("big");
+                button.classList.add("backpace");
+                button.addEventListener("click", () => {
+                    textarea.setRangeText('', textarea.selectionStart - 1, textarea.selectionEnd);
+                    textarea.focus();
+                  });
+            }else if(russianKeys[key] == "CapsLock"){
+                button.classList.add("big");
+                button.classList.add("capslock");
+            }else if(russianKeys[key] == "Shift"){
+                button.classList.add("big");
+                button.classList.add("shift");
+                button.addEventListener("click", () => {
+                    shift = !shift;
+                    upperCase(lang)
+                    textarea.focus();
+                })
+            }else if(russianKeys[key] == "Enter"){
+                button.classList.add("big");
+                button.classList.add("enter");
+                button.addEventListener("click", () => {
+                    textarea.value += "\n";
+                    textarea.focus();
+                  });
+            }else if(russianKeys[key] == "Tab"){
+                button.classList.add("big");
+                button.classList.add("tab");
+            }else if(typeof(russianKeys[key]) == "number"){
+                button.classList.add("number");
+            }else if(russianKeys[key] == "Space"){
+                button.classList.add("space");
+                button.addEventListener("click", () => {
+                    textarea.value += " ";
+                    textarea.focus();
+                })
+            }else if(russianKeys[key] == "Control"){
+                button.classList.add("control");
+            }else if(russianKeys[key] == "Delete"){
+                button.classList.add("delete");
+                button.addEventListener("click", () => {
+                    textarea.setRangeText('', textarea.selectionStart, textarea.selectionEnd + 1);
+                    textarea.focus();
+                  });
+            }else if(russianKeys[key] == "Alt"){
+                button.classList.add("alt");
+            }else{
+                button.classList.add("key");
+                button.addEventListener("click", () => {
+                    textarea.value += button.innerHTML; 
+                    textarea.focus();
+                })
+                
+            }
+            button.innerHTML = `${russianKeys[key]}`
+            fragment.append(button);
+        }
+        
+    }
+    keyBoard.append(fragment);
+}
+
+drawButtons(lang)
+
+const upperCase = (l) => {
+    keyBoard.innerHTML = "";
+    const fragment = document.createDocumentFragment();
+    if(l == "eng"){

```

Comment:
`switch() case ...` is best choice here and [here](https://github.com/VadimRomanovski/virtual-keyboard/pull/2/files#diff-9a9569e9d73f33740eada95275da7f30R286)